airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/shared/ActivityConfiguration.java

```diff
@@ -43,6 +43,11 @@ public class ActivityConfiguration {
       .setHeartbeatTimeout(TemporalUtils.HEARTBEAT_TIMEOUT)
       .build();
 
+ public static final ActivityOptions CHECK_ACTIVITY_OPTIONS = ActivityOptions.newBuilder()
+     .setScheduleToCloseTimeout(Duration.ofMinutes(5))
```

Comment:
1 hour seemed like a really long time for a CHECK command.  Shrunk to 5 minutes and made the ActivityConfiguration shared.